spec/models/institute_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Institute, type: :model do
+  subject { institute }
+
+  context 'when all fields contain valid values' do
+    let(:institute) { build(:institute,
+      name: 'Absolutely Awesome Academy') }
+    
+    it { is_expected.to be_valid }
+  end
+
+  context 'when fields have invalid values' do
+    let(:institute) { build(:institute) }
```

Comment:
Create the test model in the let block

```suggestion
    let(:institute) { build(:institute, name: '') }
```

Reply:
Yeah, fair point - this was based on the `course_spec` which has multiple checks for handling of valid/invalid fields as suggested in the last task. Since we're only doing one check here, it can be simplified as you mention - I'll update it.